### ISSUE TITLE
Hyperkit support

### ISSUE THREAD

**Issue by JesusTheHun** [codekitchen/dinghy#295]
`hyperkit` seems to have become the future of `xhyve`.
According to https://github.com/machine-drivers/docker-machine-driver-hyperkit : 

> The Hyperkit driver will eventually replace the existing xhyve driver and uses moby/hyperkit as a Go library.

Do you think it's possible for you to add the support of `hyperkit` in a remotely near future ?

**Comment by codekitchen** [comment]
I wasn't familiar with this new driver, thanks for bringing it to my attention. But to be honest I'm a bit confused by that repo -- it hasn't seen any activity in almost two years, and some of the open issues and PRs sound critical to getting it into an actual usable state.

Adding support to Dinghy should in theory be very simple but I don't think I would bother until the driver is stabilized more.

**Comment by JesusTheHun** [comment]
This driver is offered by minikube so I would say it's stable. They actually advertise it as a more stable version of xhyve (among other things).
I think the repo is just the docker driver interface, the actual driver is here https://github.com/moby/hyperkit

**Comment by codekitchen** [comment]
Yes, I'm talking specifically about the docker-machine driver project that you linked to, not hyperkit itself. But the docker-machine interface is what dinghy would be using, since dinghy uses docker-machine under the hood.

**Comment by JesusTheHun** [comment]
Ok.
I see this package in every doc I look into, including minishift.
But it is indeed weird that this package looks... abandoned ?

I tried to use `docker-machine create -d hyperkit` but sadly I already had issues (leading me to search for another driver) so I'm not sure if my setup is the issue (most likely) or the package.

**Comment by codekitchen** [comment]
Dang, that stinks. Let me know if you do get it working at some point.

**Comment by JesusTheHun** [comment]
To clarify I had issues with `xhyve` and Big Sur, I think it broke things.
So I used a fork that has PR merged ( https://github.com/zazula/docker-machine-driver-hyperkit ) and followed the few instructions, and it works great ! 

I created the machine directly with `docker-machine` and dinghy can start and stop it without trouble. It's only been a few hours but so far so good :)

**Comment by codekitchen** [comment]
Cool yeah, dinghy doesn't actually care which driver you're using once the machine is created, it just needs to know in order to pass the correct create options. So that'll work great as a workaround. Hopefully that PR will get merged in at some point.

**Comment by JesusTheHun** [comment]
Just to give a head's up : one month are everything runs just perfectly with https://github.com/zazula/docker-machine-driver-hyperkit. The original repo seems idle for a long time now.
